ccmlib/node.py

```diff
@@ -19,13 +19,16 @@
 from collections import namedtuple
 from typing import List, Optional
 
-import yaml
+from ruamel.yaml import YAML
 
 from ccmlib import common
 from ccmlib.cli_session import CliSession
 from ccmlib.repository import setup
 from ccmlib.utils.version import parse_version
 
+yaml = YAML()
```

Comment:
safe_load was used, shouldn't then add `yaml=YAML(typ='safe') `?

Reply:
maybe we could, not sure it's really critical to this project

`safe_load` was used cause pyyaml was issues warning on not using it.